frontend/js/tests/__mocks__/freshReleasesSitewideData.json

```diff
@@ -1,15 +1,16 @@
 [
     {
-        "artist_credit_name": "をとは",
+        "artist_credit_name": "\u3092\u3068\u306f",
```

Comment:
Are those unicode changes expected?

Reply:
Fixed in 2285 and rebased to include fix.